pxr/imaging/plugin/hdRpr/rprApi.cpp

```diff
@@ -2169,6 +2169,69 @@ class HdRprApiImpl {
                 RPR_ERROR_CHECK(m_rprContext->SetParameter(rpr::ContextInfo(RPR_CONTEXT_RENDER_QUALITY), hybridRenderQuality), "Fail to set context hybrid render quality");
             }
         }
+
+        if (preferences.IsDirty(HdRprConfig::DirtyHybrid) || force) {
```

Comment:
Please double check there is no code duplication here

Reply:
Removed the code duplication